src/App.js

```diff
@@ -84,7 +89,8 @@ function App() {
 
   return (
     <>
-      <Card className="player">{player1? "Gracz 1":"Gracz 2"}</Card>
+      {Winner && <WinnerWindow winner={Winner} newGame={newGameHandler} />}
```

Comment:
I dislike this option of making conditional elements, because now it's either `<WinnerWindow>` or `null`.
I would have done this as:
`Winner ? <WinnerWindow winner={Winner} newGame={newGameHandler} /> : </>`
So that it be either `<WinnerWindow>` or `<>` (i.e. both options are jsx elements I believe.

This whole comments is just an opinion tho.